gooddata-server-oauth2-autoconfigure/src/test/kotlin/AuthenticationUtilsTest.kt

```diff
@@ -512,11 +510,11 @@ internal class AuthenticationUtilsTest {
 
         private val VALID_AZURE_B2C_OIDC_CONFIG: Map<String, Any> = mapOf(
             "issuer" to "https://some-microsoft-issuer.com/someGuid/v2.0/",
-            "authorization_endpoint" to "${AZURE_B2C_ISSUER}/oauth2/v1/authorize",
-            "token_endpoint" to "${AZURE_B2C_ISSUER}/oauth2/v1/token",
-            "userinfo_endpoint" to "${AZURE_B2C_ISSUER}/oauth2/v1/userinfo",
-            "registration_endpoint" to "${AZURE_B2C_ISSUER}/oauth2/v1/clients",
-            "jwks_uri" to "${AZURE_B2C_ISSUER}/oauth2/v1/keys",
+            "authorization_endpoint" to "${AZURE_B2C_ISSUER.removeVersionSegment()}/oauth2/v1/authorize",
+            "token_endpoint" to "${AZURE_B2C_ISSUER.removeVersionSegment()}/oauth2/v1/token",
+            "userinfo_endpoint" to "${AZURE_B2C_ISSUER.removeVersionSegment()}/oauth2/v1/userinfo",
+            "registration_endpoint" to "${AZURE_B2C_ISSUER.removeVersionSegment()}/oauth2/v1/clients",
+            "jwks_uri" to "${AZURE_B2C_ISSUER.removeVersionSegment()}/oauth2/v1/keys",
```

Comment:
Wouldn't it be better to introduce a new constant, than calling the removal 9 times? :D